python/cudf_polars/cudf_polars/dsl/utils/groupby.py

```diff
@@ -71,16 +71,15 @@ def rewrite_groupby(
     unsupported.
     """
     if len(aggs) == 0:
-        # TODO: use Distinct when the partitioned executor supports it
-        return ir.GroupBy(
+        return ir.Distinct(
```

Comment:
I've confirmed that there's a test in the (non-experimental) `tests/test_groupby.py`.